src/main/java/com/example/user/adapter/inbound/controller/UserController.java

```diff
@@ -0,0 +1,23 @@
+package com.example.user.adapter.inbound.controller;
+
+import com.example.user.adapter.inbound.dto.UserSignUp;
+import com.example.user.port.inbound.UserService;
+import lombok.RequiredArgsConstructor;
+import org.springframework.web.bind.annotation.PostMapping;
+import org.springframework.web.bind.annotation.RequestBody;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RestController;
+
+import javax.validation.Valid;
+
+@RestController
+@RequestMapping("/users")
+@RequiredArgsConstructor
+public class UserController {
+  private final UserService userService;
+
+  @PostMapping(path = "/signup")
```

Comment:
@PostMapping, @GetMapping 등등 Spring MVC 의 Annotation에서 default로 어노테이션에 값을 입력 했을 시 path 경로를 받습니다.

해당 API에서 consume이나  produces, header 와 같은 다른 속성을 사용하지 않는다면 축약하는게 더 나을 것 같아요!

```suggestion
  @PostMapping("/signup")
```

Reply:
회원가입 시 signup을 명시하는 경우도 있고 POST METHOD 로만 구분하는 경우도 있는데 
어떤 걸 더 선호하시나요? -> 이것도 취향의 차이일까요~?

@PostMapping("/users/signup")
@PostMapping("/users")

Reply:
음 이것도 개개인마다 기준이 다를 순 있을 것 같은데 , 
회원 가입, 로그인, 탈퇴와 같은 API들은(특수하다고 생각되는) 행위를 명확하게 url에 표현을 합니다.

회원 시스템에서 회원이 추가되는 케이스는 얼마든지 다양해질 수 있기에(ex: 사용자 초대, 비회원, 소셜 로그인 등등)
명확하게 표현하는 것이 좋습니다.

반대로, 로그인 이력 정보, 포인트 등 보편적인 리소스 정보에 대해서는 POST /access-history , POST /points 와 같은
URL 설계로도 충분히 대응이 되기도 합니다.

Reply:
넵! 감사합니다!